src/database/Ligare/database/migrations/alembic/__main__.py

```diff
@@ -2,30 +2,30 @@
 from os import environ
 
 # this is Alembic's main entry point
-from .bl_alembic import BLAlembic
+from .ligare_alembic import LigareAlembic
 
 
-def bl_alembic(
+def ligare_alembic(
     argv: list[str] | None = None,
     log_level: int | str | None = None,
     allow_overwrite: bool | None = None,
 ) -> None:
     """
-    A method to support the `bl-alembic` command, which replaces `alembic.
+    A method to support the `ligare-alembic` command, which replaces `alembic`.
```

Comment:
This entire PR was worth it to catch this typo.

Reply:
I'll open a second PR for just the typo. ;)